docs/components/styling.md

```diff
@@ -74,10 +74,9 @@ const Button = ({ large, danger }) => {
 };
 ```
 
-_Please note: referencing classes and toggling them based on properties can become unwieldy so the [classnames] or [classcat] packages can help avoid some of the formatting hassle._
+_Please note: referencing classes and toggling them based on properties can become unwieldy so the [classcat] package can help avoid some of the formatting hassle._
```

Comment:
What did it do to you?

Reply:
oh haha i could not get it working with webpack ESM _at all_. looking at the source code it's a wonder how it ever worked in an ESM context and the maintainers seem quite hostile to writing it in a sensible way